src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/ElasticsearchCaseSearchOperation.java

```diff
@@ -110,12 +113,49 @@ private CaseSearchResult toCaseDetailsSearchResult(MultiSearchResult multiSearch
                 throw new BadSearchRequest(errMsg);
             }
             if (response.searchResult != null) {
+                buildCaseTypesResults(response, caseTypeResults, crossCaseTypeSearchRequest);
                 caseDetails.addAll(searchResultToCaseList(response.searchResult));
                 totalHits += response.searchResult.getTotal();
             }
         }
 
-        return new CaseSearchResult(totalHits, caseDetails);
+        return new CaseSearchResult(caseTypeResults, totalHits, caseDetails);
+    }
+
+    private void buildCaseTypesResults(
+        MultiSearchResult.MultiSearchResponse response,
+        List<CaseTypeResults> caseFieldsAggregations,
+        CrossCaseTypeSearchRequest crossCaseTypeSearchRequest) {
+
+        if (hitsIsNotEmpty(response)) {
+            String indexName = getIndexName(response);
+            caseFieldsAggregations.add(new CaseTypeResults(getCaseTypeIDFromIndex(indexName,
+                crossCaseTypeSearchRequest.getCaseTypeIds()),
+                response.searchResult.getTotal())
+            );
+        }
+    }
+
+    private String getIndexName(MultiSearchResult.MultiSearchResponse response) {
+        return response.searchResult.getJsonObject().getAsJsonObject("hits").get("hits")
```

Comment:
`hits` is being used quite a lot at least in this class - introduce a constant.

Reply:
DONE